tap_linkedin_ads/client.py

```diff
@@ -130,6 +130,33 @@ def check_access_token(self): #pylint: disable=inconsistent-return-statements
             else:
                 return False
 
+    @backoff.on_exception(backoff.expo,
+                          Server5xxError,
+                          max_tries=5,
+                          factor=2)
+    def check_accounts(self, config):
+        headers = {}
+        if self.__user_agent:
+            headers['User-Agent'] = self.__user_agent
+        headers['Authorization'] = 'Bearer {}'.format(self.__access_token)
+        headers['Accept'] = 'application/json'
+
+        if config.get('accounts'):
+            account_list = config['accounts'].replace(" ", "").split(",")
+            invalid_account = []
+            for account in account_list:
+                response = self.__session.get(
+                    url='https://api.linkedin.com/v2/adAccountUsersV2?q=accounts&count=1&start=0&accounts=urn:li:sponsoredAccount:{}'.format(account),
+                    headers=headers)
+                # Account users API will return 404 if provided account is valid number but invalid LinkedIn Ads account
+                if response.status_code == 404:
+                    invalid_account.append(account)
+                elif response.status_code != 200:
+                    raise_for_error(response)
+            if invalid_account:
+                error_message = 'These accounts are invalid LinkedIn Ads accounts from provided accounts:{}'.format(invalid_account)
```

Comment:
@savan-chovatiya The error message should be: "Invalid Linked Ads accounts provided during the configuration: [a1, a2. a3]"

Reply:
@karanpanchal-crest Updated error message as per suggestion.